docs/guides/creatingasource.md

```diff
@@ -0,0 +1,118 @@
+# Creating a Source
```

Comment:
In some pages you removed the title, does this page need one? (sorry, not familiar with Mkdocs)

Reply:
it looks like if there is no Title in the source then mkdocs uses what is set in the menu